### ISSUE TITLE
Missing notifications for local file changes

### ISSUE THREAD

**Issue by knstntnb** [samschott/maestral#791]
**Describe the bug**
Unfortunately, I am not receiving notifications of any kind, even though the settings in maestral are set to on. Meanwhile, the Mac settings do not show maestral under notifications and therefore are not on - maestral is not asking for permissions within the Mac environment on my end.

**To Reproduce**
(firstly, I installed maestral via homebrew, then removed it completely with an ending reboot)
downloaded maestral via GitHub, signed in and authenticated the app to get the token, sync is in progress and notifications in app settings are on while in Mac settings no app permission was requested. App settings for notification have been turned off and on again, no change.

**Expected behaviour**
Notifications would be delivered upon turning settings on + app would be listed within Mac notifications settings.

**System:**
 - Maestral version: GUI v1.6.4, daemon v1.6.4
 - OS: macOS Ventura 13.1

**Comment by samschott** [comment]
Permission will be requested not when the checkbox in Maestral's settings is checked, but when attempting to send the first notification.

Can you confirm that you did expect a notification to be sent, for example by manually creating a new folder on the Dropbox website?

Another thing to check: Notifications can be delivered silently, directly to the macOS "notification center", under some circumstances. Do you see any there?

I'm running Ventura 13.1 myself and everything appears to work as expected on my machine.

**Comment by samschott** [comment]
> However, there are no notifications upon syncing directly from the Mac to cloud

Yes, that is indeed the expected behaviour. Maybe this can be clarified in the checkbox text, for example: "Notify for remote file changes".

Would you think that notifications for local changes are helpful? I always assumed that they add little value.
